services/src/test/scala/cromwell/pipeline/service/ProjectConfigurationServiceTest.scala

```diff
@@ -1,14 +1,14 @@
 package cromwell.pipeline.service
 
 import java.nio.file.Paths
-
 import com.mongodb.client.result.UpdateResult
 import cromwell.pipeline.datastorage.dao.repository.DocumentRepository
 import cromwell.pipeline.datastorage.dao.repository.utils.TestProjectUtils
-import cromwell.pipeline.datastorage.dto.{ FileParameter, ProjectConfiguration, ProjectFileConfiguration, StringTyped }
+import cromwell.pipeline.datastorage.dto.{FileParameter, ProjectConfiguration, ProjectFileConfiguration, StringTyped}
```

Comment:
Looks like you haven't configured your IDE to use `.scalafmt.conf`. Go to `Settings -> Editor -> Code Style -> Scala`, choose `Scalafmt` formatter and set a `Project` scheme.

Reply:
fixed:)